README.md

```diff
@@ -73,10 +73,10 @@ Flags:
       --max-extraction-size=1073741824     Maximum extraction size that allowed is (in bytes). (disable check: -1)
       --max-extraction-time=60             Maximum time that an extraction should take (in seconds). (disable check: -1)
       --max-input-size=1073741824          Maximum input size that allowed is (in bytes). (disable check: -1)
+  -P, --no-preserve-file-attributes        Do not preserve file attributes (mode, modtime, access time).
   -N, --no-untar-after-decompression       Disable combined extraction of tar.gz.
   -O, --overwrite                          Overwrite if exist.
-  -P, --pattern=PATTERN,...                Extracted objects need to match shell file name pattern.
-  -p, --preserve-file-attributes           Preserve file attributes from archive (access and modification time & file permissions).
+      --pattern=PATTERN,...                Extracted objects need to match shell file name pattern.
```

Comment:
This is a breaking change, do we know if anyone is relying on this tool in their automations etc?

Reply:
As by now, I am not 100% sure if anyone using it in the CLI. As by https://github.com/hashicorp/go-extract/pull/106#discussion_r1889817092 I will revoke this breaking change and add the capital `-P` flag back.

Reply:
I revoked that breaking change in https://github.com/hashicorp/go-extract/pull/106/commits/5ff62798be4f469be5c42b1c77c3a23f40ca5a5d.

Reply:
OK, thanks, it might be ok to make breaking changes sometimes, but now this is open source we have no way to know how it's being used, so I would tend to be conservative with things like this.